contrib/go-pg/pg/example_test.go

```diff
@@ -0,0 +1,33 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package pg_test
+
+import (
+	"context"
+	"log"
+
+	pggotrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/go-pg/pg"
+
+	"github.com/go-pg/pg/v10"
+)
+
+// To trace Postgres queries, wrap pg.Connect instance with pggotrace.Hook.
```

Comment:
Please expand this comment slightly, such as `To trace Postgres queries, create a connection with pg.Connect, and  then pass it to pggotrace.Hook`.